dist/api_data.py

```diff
@@ -239,6 +239,17 @@ def __str__(self):
                             the number of triggers for which to wait''',
                             min='1', max='4294967295'),
                 ]),
+            ConnectionControlPoint('thread_wait_for_upd_abort', 'Trigger', 'Always', '', 
```

Comment:
Please add above it:
    # For test/suite/test_bug035.py and test/suite/bug036.py